.github/workflows/workflow.yml

```diff
@@ -193,8 +193,6 @@ jobs:
         run: sudo npm install -g markdown-spellcheck
       - name: Check Docs links and IPFS hashes
         run: tox -e check-doc-links-hashes
-      - name: Check API Docs updated
```

Comment:
Why removed?

Reply:
Sorry I forgot about this one, basically the doc check was failing and only way to fix it was to generate docs with new mkdocs package which would have introduced a lot of new files and since we want to avoid introducing too much changes in the base PR I skipped this check temporarily. @dvilelaf will re enable this in a child PR and generate the latest API docs

Reply:
Ok, I'll add it in the linter PR